doc: avoid 'simply' (#266434)
NixOS/nixpkgs: nixos/doc/manual/configuration/modularity.section.md

[](#opt-environment.systemPackages). When multiple modules define an
option, NixOS will try to *merge* the definitions. In the case of
[](#opt-environment.systemPackages), that's easy: the lists of

> packages can be concatenated. The value in `configuration.nix` is

Reviewer: ```suggestion
packages are just concatenated. The value in `configuration.nix` is
```

Reviewer: I think it becomes less readable without "simply" or an equivalent.

Author: I don't know, still seems noisy to me. I tried making it more succinct, WDYT?

Reviewer: Yes, better. The original wording would have made sense if it then showed another example of a type with complex merging rule, but like this is kind of pointless.
